libmscore/score.cpp

```diff
@@ -2302,113 +2302,124 @@ void Score::splitStaff(int staffIdx, int splitPoint)
       int strack = staffIdx * VOICES;
       int dtrack = (staffIdx + 1) * VOICES;
 
+      // Keep track of ties to be reconnected.
+      struct OldTie {
+            Tie* tie;
+            Note* nnote;
```

Comment:
I recommend always initializing pointers to nullptr, make it a habit 
   
```
struct OldTie {
            Tie* tie{nullptr};
            Note* nnote{nullptr};
}
```

Reply:
Wouldn't 

      struct OldTie {
            Tie* tie {};
            Note* nnote {};

do the same?

Reply:
Yes, it is same. I prefer {nullptr}, because it’s more explicit, but you can just {}

Reply:
Initialising is better indeed. Also, when initialising, I prefer <code>{ nullptr }</code> above a plain <code>{ }</code>, I don't implicit things that much.

Reply:
initializing needed

Reply:
Is in this case not possible because
1. In line 2366 a new <code>oldTie</code> structure is create using uniform initialization,
2. On Linux (and BTW also Apple) the we still use C++11.
3. In C++11 this uniform initialization is not compatible with non-static member initialization.
So, in order to do this initialization (which is my preference) we should upgrade the compiler on Linux/Apple to C++14.